CHANGES

```diff
@@ -1,6 +1,8 @@
-Upcoming release (1.0.0)
+1.0.0 (January 24, 2018)
```

Comment:
Should this be 25 now?

Reply:
@effigies I've updated it for [the official release](https://github.com/nipy/nipype/releases/tag/v1.0.0)